README.md

```diff
@@ -131,6 +142,70 @@ The `upgrade` command upgrades an existing helm release. A config file has to be
 
 The `version` command pretty much does what it says on the tin: it gives out the version currently used.
 
+<a name="config"></a>
+## Configuration
+
+A `havener config file` provides an easy solution for configurating and deploying one or multiple Helm Charts. The `config` is saved as a `.yml-file` and is used by the `deploy` and `upgrade` commands. Besides information about the charts, it can overide the values.yml file and can contain further pre- and post-processing steps.
```

Comment:
you can use `yaml` file instead of `.yml-file` . 
You have a typo in `overide`, it should be `override`

Reply:
Or, just replace "a `.yml-file`" with "a YAML file".

Reply:
Should be named `values.yaml` instead of `values.yml`.